typing_extensions/src_py3/test_typing_extensions.py

```diff
@@ -1135,12 +1148,10 @@ def meth(self):
                 self.assertFalse(P._is_runtime_protocol)
             self.assertTrue(PR._is_runtime_protocol)
             self.assertTrue(PG[int]._is_protocol)
-            self.assertEqual(P._get_protocol_attrs(), {'meth'})
-            self.assertEqual(PR._get_protocol_attrs(), {'x'})
-            self.assertEqual(frozenset(PG._get_protocol_attrs()),
-                             frozenset({'x', 'meth'}))
-            self.assertEqual(frozenset(PG[int]._get_protocol_attrs()),
-                             frozenset({'x', 'meth'}))
+            self.assertEqual(typing_extensions._get_protocol_attrs(P), {'meth'})
+            self.assertEqual(typing_extensions._get_protocol_attrs(PR), {'x'})
+            self.assertEqual(frozenset(typing_extensions._get_protocol_attrs(PG)),
+                                 frozenset({'x', 'meth'}))
```

Comment:
Did you intentionally delete the test for `PG[int]`?

Reply:
Yes, but I think it is better to keep it under `if not PEP_560:`.